ui-tests/cypress/integration/pages/templates.spec.js

```diff
@@ -1,142 +1,621 @@
-import HomePage       from '../../support/pageObjects/HomePage';
-import {htmlElements} from '../../support/pageObjects/WebElement';
-
-const sampleData = {
-  code: '2-2_democol',
-  descr: '2-2 Demo Col',
-  configuration: '{"frames": [{"pos": 0, "descr": "Logo", "mainFrame": false, "defaultWidget": {"code": "logo","properties": null}, "sketch": {"x1": 0, "y1": 0, "x2": 2, "y2": 0}}]}',
-  template: '<h1>hello there a frame</h1>'
-};
-
-const openPageTemplateMgmtPage = () => {
-  let currentPage = new HomePage();
-  currentPage     = currentPage.getMenu().getPages().open();
-  currentPage     = currentPage.openTemplates();
-  cy.wait('@loadedTemplatesList');
-  return currentPage;
-};
-
-describe([Tag.GTS], 'Page Templates', () => {
+import TemplatesPage      from "../../support/pageObjects/pages/templates/TemplatesPage";
+import {htmlElements}     from "../../support/pageObjects/WebElement";
+import {generateRandomId} from "../../support/utils";
+
+describe('Page Templates', () => {
+
+  const sampleData = {
```

Comment:
We can extract this const objects to fixture as json files instead of having them as const in the spec file